00 DART Core (Required)/gamedata/scripts/DART.script

```diff
@@ -7,6 +7,11 @@ aspectRatio21x9 = 21 / 9
 aspectRatio32x9 = 32 / 9
 enableLogging = true
 
+-- shared globals
+DART.HUD_SAFE_ZONE = 0
+DART.hud_health_system_y = 0
+DART.alticons_x = 0
+
```

Comment:
Ran into some trouble setting globals that were usable in other scripts. Given the load order is alphabetical, i set defaults here then changed them later in scripts. I do not know if this is the best way to handle data across scripts but i found it to be working fine in testing

Reply:
I think I understand -- because you want to set the values in one module and read them in another? Yeah I'll think about it but I think it's fine for now.

My first reaction was, it'd be great if we could avoid it so that the news/messages module (for example) wouldn't depend on the BHS module being active. But then I realized if the BHS isn't active, this is all moot anyway because there'd be no scale to worry about.

So yeah I commented the other day about being wary of making everything intertwined vs. just letting people manually move stuff, but I'm starting to warm up to it.